src/main/java/lotto/domain/Ranking.java

```diff
@@ -25,7 +26,11 @@ public Integer getPrize() {
         return prize;
     }
 
-    public static Ranking find(int count) {
+    public static Ranking find(int count, boolean hasBonusBall) {
+        if (count == Ranking.SECOND_BONUS.getCount() && !hasBonusBall) {
```

Comment:
`Ranking.SECOND` 을 return 하기위한 if 조건이면 
```suggestion
        if (count == Ranking.SECOND.getCount() && !hasBonusBall) {
```
이렇게 작성해주시는게 가독성이 더 좋을 것 같아요

Reply:
감사합니다!